src/Aspire.ProjectTemplates/templates/aspire-starter/9.1/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -17,11 +17,11 @@
     <PackageReference Include="Aspire.Hosting.Testing" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
     <PackageReference Include="coverlet.collector" Version="6.0.2" Condition=" $(TestFramework) == 'xUnit.net' " />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" Condition=" $(TestFramework) != 'MSTest' " />
-    <PackageReference Include="MSTest" Version="3.4.3" Condition=" $(TestFramework) == 'MSTest' " />
+    <PackageReference Include="MSTest" Version="3.6.1" Condition=" $(TestFramework) == 'MSTest' " />
     <PackageReference Include="NUnit" Version="4.1.0" Condition=" $(TestFramework) == 'NUnit' " />
     <PackageReference Include="NUnit.Analyzers" Version="4.2.0" Condition=" $(TestFramework) == 'NUnit' " />
     <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" Condition=" $(TestFramework) == 'NUnit' " />
-    <PackageReference Include="xunit" Version="2.9.0" Condition=" $(TestFramework) == 'xUnit.net' " />
+    <PackageReference Include="xunit" Version="2.9.2" Condition=" $(TestFramework) == 'xUnit.net' " />
```

Comment:
Same as above but for the XUnit.net templates in the .ENT SDK 9.0.200

Reply:
it matches

Reply:
btw the older templates use 2.9.0 and 2.9.2 in a mix. too late now.